src/org/jitsi/impl/neomedia/RTPConnectorOutputStream.java

```diff
@@ -768,11 +768,19 @@ public long getOutputBitrate(long now)
          */
         final Thread sendThread;
 
+        QueueStatistics queueStats = null;
```

Comment:
Added javadoc. I didn't use the "private" keyword, because these are members of a private inner class, so they cannot be accessed from the outside (unless I am missing something).